src/picongpu/include/plugins/radiation/Radiation.hpp

```diff
@@ -259,8 +259,8 @@ class Radiation : public ISimulationPlugin
 private:
 
     /**
-     * The plugin is loaded on every host pc, and therefor this function is
-     * executed on every host pc.
+     * The plugin is loaded on every host computer, and therefor this function is
```

Comment:
"host computer" is a node or even more precise: "each MPI rank", because it's loaded multiple times per node.